docs/content/v2024.1/benchmark/sysbench-ysql.md

```diff
@@ -37,55 +67,63 @@ Make sure you have the [YSQL shell](../../api/ysqlsh/) `ysqlsh` exported to the
 $ export PATH=$PATH:/path/to/ysqlsh
 ```
 
-### Start YugabyteDB
+  </div>
 
-Start your YugabyteDB cluster by following the steps in [Manual deployment](../../deploy/manual-deployment/).
+  <div id="rhel" class="tab-pane fade" role="tabpanel" aria-labelledby="rhel-tab">
 
-{{< tip title="Tip" >}}
-You will need the IP addresses of the nodes in the cluster for the next step.
-{{< /tip>}}
+```sh
+wget https://github.com/yugabyte/sysbench/releases/download/1.0.0-yb/sysbench-1.0.0-1.el8.x86_64.rpm
 
-### Run the benchmark
+sudo yum install -y sysbench-1.0.0-1.el8.x86_64.rpm 
+```
 
-Run the `run_sysbench.sh` shell script to load the data and run the various workloads:
+  </div>
+  <div id="macos" class="tab-pane fade" role="tabpanel" aria-labelledby="macos-tab">
 
 ```sh
-./run_sysbench.sh --ip <ip>
+brew install postgresql@14 wget
+
+wget https://github.com/yugabyte/sysbench/releases/download/1.0.0-yb/Sysbench.pkg
+
+sudo  installer -pkg Sysbench.pkg -target /
 ```
 
```

Comment:
where is sysbench installed?

Reply:
Can you check the updated files? It does mention the installation.

Reply:
or did you meant by the location or files folder? cc @shaharuk-yb

Reply:
sysbench gets installed as a cli binary and is accessible from any path(as after installation it is in /usr/local/bin/)